acme-stack/src/main/packaging/conf/application.conf

```diff
@@ -14,47 +14,154 @@ modules = [
   ########### Forms ###########
   "forms=io.knotx.forms.core.FormsKnot"
   ########### Custom ###########
-  "formsAdapter=com.acme.forms.adapter.http.FormsAdapterVerticle"
+  "customFormsAdapter=com.acme.forms.adapter.http.FormsAdapterVerticle"
   "reqProcessorKnot=com.acme.gateway.RequestProcessorKnotVerticle"
   "serviceMock=com.acme.content.MockServiceVerticle"
 ]
 
-########### Globals ###########
 global {
-  serverPort = 8092
 
-  snippetTagName = script
+  ########### Snippets ###########
+  snippetOptions {
+    # Knot.x snippet HTML tag name. Default is 'script'.
```

Comment:
Just to remember (to-do in another PR): `knotx:snippet` will be default since `2.0`.

Reply:
colon is problematic, why not `knotx-snippet`  ? colon indicates a xml namespace for some IDE's and causes IDE warnings in the end

Reply:
`knotx` will be a separate namespace of Knot.x tags. There will be more types than just `knotx-snippet`. Warnings in the IDE is not a real problem (you can configure IDE for that), sorry :)

Reply:
it was not only a problem on IDE as I remember. it was impossible to use it in HTL templates.

Reply:
@pan-ky started discussion about HTL on separate Slack channel